src/AsyncGenerator.yml

```diff
@@ -150,9 +150,6 @@
   transformation:
     configureAwaitArgument: false
     localFunctions: true
-    asyncLock:
```

Comment:
I will most likely remove this option in the future as adding an additional field for async locking allows two threads to execute the "same code" simultaneously.